src/fetch.rs

```diff
@@ -272,8 +273,20 @@ fn get_default_timeout() -> Duration {
     Duration::from_secs(10)
 }
 
+// workaround from rustup
+fn env_proxy(url: &reqwest::Url) -> Option<reqwest::Url> {
+    env_proxy::for_url(url).and_then(|(host, port)| {
+        // TODO: update when inejge/env_proxy/pull/3 is merged
```

Comment:
If only this was a full link, I could click it.

And if only someone would mention this pull request in a comment, Github would link to it…

https://github.com/inejge/env_proxy/pull/3